src/util/animation/Animation.ts

```diff
@@ -0,0 +1,30 @@
+import { AnimationBase } from './AnimationBase';
+import { AnimationOptions } from './types';
+
+export class Animation extends AnimationBase<number> {
```

Comment:
Can we call this ValueAnimation? If we have ArrayAnimation we need a name that is a sibling of ArrayAnimation.
`Animation` only isn't clear enough.

Reply:
sure
makes sense

Reply:
done